packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -78,6 +78,23 @@ const TutorialStore = types
 
       if (miniCourse) return lastStepSeen === miniCourse.steps.length - 1
 
+      return false
+    },
+
+    get isFirstStep () {
+      if (!!self.active && !!self.activeStep.toString()) {
```

Comment:
Why the `!!` here? I thought the if statement would automatically treat the conditionals as booleans? Is it important to actually cast them to Boolean?

Reply:
Oh wait, is this catching a case where `self.activeStep.toString()` returns '0' and JavaScript evaluates that as false? I'm confused as to what the conditional is actually checking, and why `self.active` isn't a boolean by default.

Reply:
Yes, you're correct about the '0' issue in javascript. I need it to treat '0' as truthy in this case. However, I don't remember why I was casting `self.active` to boolean here. Let me try removing it... but I think I vaguely remember doing it because I found a scenario where it could fail in a test without doing this. I'll confirm and can leave a comment when I re-figure it out again.

Reply:
I can't determine why I did that, so I'll remove it from `self.active`